02.explore/02.personalize-experience/03.install-apps/docs.md

```diff
@@ -4,4 +4,72 @@ taxonomy:
     category: docs 
 ---
 
-* Talk about how apps let you get more out of your experience – like displaying names, fingerpainting, or taking pictures with a camera
\ No newline at end of file
+You can enhance your experience in High Fidelity by installing apps from the Marketplace. These apps enable you to get more out of High Fidelity. Do you want to record your avatar dancing, take selfies, or finger paint? Or are you looking to livestream to YouTube or make your avatar clap? Our Marketplace has a number of apps that can help you customize your experience in-world. 
+
+On this page:
+
+* [Install an App]($install-an-app)
+* [Our Marketplace Apps](#our-marketplace-apps)
+
+  * [Spectator Cam: Record or Livestream in High Fidelity](#spectator-cam)
+
+  * [Finger Painting](#finger-painting)
+
+
+## Install an App
+
+We and many users have created apps that are available in High Fidelity's Marketplace. To find and install an app:
+
+*  Go to **Market** on your HUD or Tablet. and click 'Categories'.
+*  Select 'Apps, Scripts, & Tools' from the dropdown to take a look at what's available. You can also type 'app' in the search bar. Apps can be free or may be purchased at the listed price. If an app is not free, set up your [Wallet](#set-up-your-wallet) before purchasing anything on the Marketplace.
+*  Click the app of your choice to purchase it. Once you've purchased your app, you can view it in your **Inventory** or **Recent Activity**. 
+*  Click **Install App**.
+*  Once the app is installed, click **Open App** to launch. You can also access the app through your HUD or Tablet.
+
+## Our Marketplace Apps
+
+We've listed some of the apps we created and how you can use them. 
+
+### Spectator Cam
+
+The Spectator Camera is a camera you can use, along with recording software such as [Open Broadcaster Software (OBS)](https://obsproject.com/), to record or livestream what you and your friends do in High Fidelity. 
+
+>>>>> You can use the Spectator Camera only while using an HMD.
+
+You can record or livestream what you see through your Spectator Camera using OBS. Check out their [official overview guide](https://obsproject.com/forum/threads/official-overview-guide.402/) for more details.
+
+#### Get the Spectator Camera
```

Comment:
I would remove this section, and just use a sentence like "[Use the instructions above](#install-apps) to search for and install the Spectator Camera."